src/cmd-coreos-prune

```diff
@@ -112,7 +112,7 @@ def main():
         builds_json_data = json.load(f)
     # Original list of builds
     builds = builds_json_data["builds"]
-    pruned_build_ids = []
+    builds_to_prune = []
```

Comment:
maybe we should make this a set so we can't add an entry more than once, which is what I think the below code will be doing.

Reply:
True that, updating.